### PR TITLE
Allow UAC prompt

### PR DESCRIPTION
**Pull request by ghost**
This is the result of an idea during the installation and some research of the Inno documentation.

Kudos for the incredible software!

### CODE REVIEW

File: inno_setup.iss [hunk 1]
```diff
@@ -20,12 +20,12 @@ AppPublisherURL={#AppURL}
...
 AppUpdatesURL={#AppURL}
 Compression=lzma
+; Allow selection of installation with elevation or not.
+PrivilegesRequiredOverridesAllowed=commandline dialog
 DefaultDirName={commonpf}\{#AppName}
 DefaultGroupName={#AppName}
 AllowNoIcons=yes
 LicenseFile=LICENSE.txt
-; Remove the following line to run in administrative install mode (install for all users.)
-PrivilegesRequired=lowest
```
Comment (ghost) [inline inno_setup.iss:28]: Results in a Windows Error of Access Denied without proper elevation with the default folder setup.

Comment (acidicoala) [reply inno_setup.iss:28]: Hey, thanks for the catch, totally forgot that default folder (Program Files) requires admin rights 😅.
Have you been able to compile the installer with those settings and test it?

Comment (ghost) [reply inno_setup.iss:28]: I just downloaded Inno setup, I'll try it out, though I think I got it wrong

Comment (acidicoala) [reply inno_setup.iss:28]: Alright, I will test the changes tomorrow as well.

Comment (ghost) [reply inno_setup.iss:28]: Turns out i did got it wrong

Comment (ghost) [reply inno_setup.iss:28]: I'll read some docs and make another pull request when it works!

Comment (ghost) [reply inno_setup.iss:28]: There we go!

**Pull request closed without merging**